src/github.com/getlantern/flashlight/util/common.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	"errors"
+	"github.com/getlantern/golog"
+	"github.com/mitchellh/go-homedir"
```

Comment:
Ah, that's right.. we switched back to gonative. Okay, I went ahead and reverted that function to use user.Current().HomeDir instead.